SHMEM/isx.c

```diff
@@ -627,37 +622,51 @@ static void log_times(char * log_file)
       print_run_info(fp);
       print_timer_names(fp);
     }
-    print_timer_values(fp);
+    fclose(fp);
+  }
 
-    report_summary_stats();
+  for(int i = 0; i < NUM_PES; ++i) {
```

Comment:
I suppose you should get rid of the compiler warning by doing this:
```suggestion
  for(int i = 0; i < shmem_n_pes(); ++i) {
```